### ISSUE TITLE
Feature Request: allow for 5-string banjo tablature

### ISSUE THREAD

**Issue by jeremy9959** [paulrosen/abcjs#1021]
Tablature for the five string banjo is written with the high drone string at the bottom, followed in ascending order by the strings in low-to-high order.  For example, for standard open G tuning, the notes are gDGBd.  This is incompatible with the abcjs tab rule for tunings that requires the strings in order of pitch.

Being able to produce banjo tab would be great!

**Comment by paulrosen** [comment]
Yes! This is definitely been gnawing at me. I will look at this sometime this weekend and give you a recommendation.
